website/blog/2021-01/01/_index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Happy anniversary Gardener! Three years of open source Kubernetes management
+linkTitle: Happy anniversary Gardener! Three years of open source Kubernetes management
+newsSubtitle: January 30, 2021
+type: blog
+publishdate: 2021-01-25
+archivedate: 2021-02-25
+authors:
+- name: Vasu Chandrasekhara
+  email: vasu.chandrasekhara@sap.com
+  avatar: https://avatars.githubusercontent.com/u/11454395?s=400&u=9496275f0718d81bea32068dccb4d9c2d848e592&v=4
+- name: Tim Usner
+  email: tim.usner@sap.com
+  avatar: https://avatars.githubusercontent.com/u/40451181?s=400&u=4df34635cf86b924700ef4152ec8462eeaa35721&v=4
+aliases: ["/blog/2021/01/30/01"]
+---
+
+Happy new year Gardeners!
+As we greet 2021, we also celebrate Gardener’s third anniversary. 
+Looking back, three years down the line, the project initiators were working towards a special goal: Publishing Gardener as an open source project on Github.com.
+
+Join us as we look back at how it all began, the challenges Gardener aims to solve and why open source and the community was and is the project’s key enabler.
+
+## Gardener kick-off: “We opted to BUILD ourselves”
+
+Early 2017, SAP put together a small, jelled team of experts with a clear mission: work out  how SAP could serve Kubernetes based environments (as a service) for all teams within the company.  Later that same year SAP also joined the [CNCF](https://www.cncf.io/) as platinum member.
+
+We first deliberated intensively on the BUY options. There were some early products from commercial vendors and startups available that did not bind exclusively to one of the hyperscalers. But these products did not cover many of our crucial and immediate requirements for a multi-cloud environment.
```

Comment:
At the size and envisioned volume of SAP - it would have been cheaper to buy those companies than to actually be a customer of them. You might want to reflect that in.